app/jenkins-for-jira-ui/src/App.tsx

```diff
@@ -70,7 +82,10 @@ const App: React.FC = () => {
 			<Router history={history}>
 				<Switch>
 					<Route exact path="/">
-						<JenkinsServerList />
+						{renovateConfigFlag
+							? <MainPage />
```

Comment:
Decided to call this 'main page' instead of config/admin page as the content for both the admin and global pages will effectively be exactly the same as both. Only diff is that on the global page we'll be hiding anything that only an admin should be able to do (connecting servers, deleting etc). So need a name that is generic and works for both cases. Open to suggestions is main page isn't vibing with ya.

Reply:
I'm a bit confused on this because I haven't seen the designs. When I see "Main" I assume that it's the main page on the app. If it's a wrapper on the content, I'd add the word "Container" to signify that. Not fussed either way though - will defer to your judgement

Reply:
Yeah I'm not super sold on this one. I'll add it to my list of clean-up items and ponder on it a little more.